fixtures/nodejs-hybrid-app/src/index.ts

```diff
@@ -29,8 +28,6 @@ export default {
 				return testX509Certificate();
 			case "/test-require-alias":
 				return testRequireUenvAliasedPackages();
-			case "/test-unenv-preset":
-				return await testUnenvPreset();
```

Comment:
Given that the testing strategy is now to test local changes to the unenv preset in the package, it might be worth keeping this test here so that we also get test coverage on the published unenv preset?

Reply:
I think it makes sens to have preset related tests in the `@cloudflare/unenv-preset` and in a single location (I don't think we will think about updating 2 places especially because they should not be updated at the same time)

An improvement I want to do (as a follow up) is to test both the stable and dev versions from the preset package. This is why I asked James and Yagiz about figuring out the workerd version at runtime (to only run applicable tests). It sounds like it is not possible but we can still pass the version via the request.